packages/nc-gui/components/smartsheet/grid/GroupByTable.vue

```diff
@@ -136,6 +136,14 @@ const pagination = computed(() => {
     extraStyle: `margin-left: ${scrollBump.value}px;`,
   }
 })
+
+async function deleteSelectedRowsWrapper() {
+  if (!deleteSelectedRows) throw new Error('Feature not implemented')
+
+  await deleteSelectedRows()
+  // reload table data
+  await reloadTableData({ shouldShowLoading: true })
```

Comment:
What is the expected UX here @dstala, throwing will cause error modal to be shown and make UI unusable until refresh if I am not missing something

Reply:
It should not throw. the `deleteSelectedRows` should always be set. The check is there just to get rid of TS warning.

Reply:
Please use following as this is the common pattern we follow
```
if (!deleteSelectedRows) {
  return
}
```